sdcm/nemesis.py

```diff
@@ -834,30 +888,39 @@ class StopStartMonkey(Nemesis):
     def disrupt(self):
         self.disrupt_stop_start_scylla_server()
 
+
+@run_in_mixed_mode
 class RestartThenRepairNodeMonkey(Nemesis):
 
     @log_time_elapsed_and_status
     def disrupt(self):
         self.disrupt_restart_then_repair_node()
 
+
+@run_in_mixed_mode
 class MultipleHardRebootNodeMonkey(Nemesis):
 
     @log_time_elapsed_and_status
     def disrupt(self):
         self.disrupt_multiple_hard_reboot_node()
 
+
+@run_in_mixed_mode
```

Comment:
This nemesis can run also on spot, no need the decorator.

Reply:
remove